homeassistant/components/sensor/nmbs.py

```diff
@@ -0,0 +1,207 @@
+"""
+Get ride details and liveboard details for NMBS (Belgian railway).
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nmbs/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'NMBS'
+DEFAULT_NAME_LIVE = "NMBS Live"
+
+DEFAULT_ICON = "mdi:train"
+DEFAULT_ICON_ALERT = "mdi:alert-octagon"
+
+CONF_STATION_FROM = 'station_from'
+CONF_STATION_TO = 'station_to'
+CONF_STATION_LIVE = 'station_live'
```

Comment:
What does this signify?

Reply:
The station to monitor live departures from...